modules/utxo-lib/src/bitgo/signature.ts

```diff
@@ -191,20 +194,25 @@ export function parseSignatureScript(
       throw new Error('expected 2 pubkeys');
     }
 
+    const signatures = [sig1, sig2].map((b) => {
+      if (Buffer.isBuffer(b)) {
+        return b;
+      }
+      throw new Error(`unexpected signature element ${b}`);
+    }) as [Buffer, Buffer];
+
+    const scriptPathLevel = controlBlock.length === 65 ? 1 : controlBlock.length === 97 ? 2 : undefined;
```

Comment:
Should we throw in the `undefined` case here? If we have a control block, it ought to be either 65 or 97 bytes or else something is very wrong.

Reply:
let's do that